src/js/containers/search/visualizations/rank/RankVisualizationWrapperContainer.jsx

```diff
@@ -202,6 +204,11 @@ export class RankVisualizationWrapperContainer extends React.Component {
                 linkSeries.push(recipientLink);
             }
 
+            if (this.state.scope === 'awarding_agency') {
```

Comment:
Should this also include " && result._agencySlug " ? Not all agencies will have one, and that's how we know they should have a link to their page or not.

Reply:
My mistake; awarding agencies will always have slugs (I'm told).